suite-native/device-manager/src/components/WalletList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useSelector } from 'react-redux';
+
+import { selectDevice, selectDevices } from '@suite-common/wallet-core';
+import { VStack } from '@suite-native/atoms';
+import { AcquiredDevice } from '@suite-common/suite-types';
+import { getDeviceInstances } from '@suite-common/suite-utils';
+
+import { WalletItem } from './WalletItem';
+
+type WalletListProps = {
+    onSelectedDevice?: (device: AcquiredDevice) => void;
```

Comment:
Please make this prop required. Why would it be undefined?
```suggestion
    onSelectedDevice: (device: AcquiredDevice) => void;
```

Reply:
5825a46c34